app/controllers/submissions_controller.rb

```diff
@@ -64,12 +64,17 @@ def upload_file_submission(task, upload)
   end
 
   def template_for_submission(submission)
+    case (status = submission.status)
+    when 'pending' then 'processing'
+    when 'management_charge_calculation_failed' then 'errors/internal_server_error'
+    else status
+    end
+  end
+
+  def status_for_submission(submission)
     case submission.status
-    when 'pending', 'processing' then :processing
```

Comment:
what's the effect of removing these existing statuses from `template_for_submission`?

Reply:
`template_for_submission` has not changed behaviourally  - for most statuses, it just returns the name of the status, and then for a few statuses we return something different. I changed the way it was implemented because adding a new case to the that huge `case` statement caused Rubocop to emit a cyclomatic complexity error.